generation/WinSDK/enums.json

```diff
@@ -31894,6 +31894,298 @@
     },
     "members": [],
     "uses": []
+  },
+  {
+    "type": "ushort",
+    "flags": true,
+    "name": "DISPATCH_FLAGS",
+    "autoPopulate": {
+      "filter": "DISPATCH_",
+      "header": "OleAuto.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "method": "IDispatch::Invoke",
```

Comment:
Didn't know you could do this. Looks like it works, but typically we define "interface" and "method" separately for these scenarios. Would recommend doing that for consistency.